From 74645caeb73c724394ca67349d518fa09307f975 Mon Sep 17 00:00:00 2001
From: Liang Chen <cl@rock-chips.com>
Date: Fri, 2 Feb 2018 19:19:51 +0800
Subject: [PATCH] arm64: dts: rockchip: add parameters in CommandLine for
 px30-android

Some parameters in rk-parameter are not included in gpt mode, so we
need move these parameters into CommandLine.

Change-Id: Ia17d7bfc8841531c0cce0d257ff12efdef9f4c64
Signed-off-by: Liang Chen <cl@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30-android.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/px30-android.dtsi b/arch/arm64/boot/dts/rockchip/px30-android.dtsi
index 0d80981372c1..c6d37ef8cdd7 100644
--- a/arch/arm64/boot/dts/rockchip/px30-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/px30-android.dtsi
@@ -6,7 +6,7 @@
 
 / {
 	chosen {
-		bootargs = "earlyprintk=uart8250,mmio32,0xff160000 swiotlb=1";
+		bootargs = "earlyprintk=uart8250,mmio32,0xff160000 swiotlb=1 console=ttyFIQ0 androidboot.baseband=N/A androidboot.selinux=permissive androidboot.veritymode=/dev/block/by-name/metadata androidboot.hardware=rk30board androidboot.console=ttyFIQ0 init=/init";
 	};
 
 	fiq-debugger {
-- 
2.35.3

